business/workloads.go

```diff
@@ -1161,7 +1161,7 @@ func (in *WorkloadService) fetchWorkloadsFromCluster(ctx context.Context, cluste
 
 		// Add the Proxy Status to the workload
 		for _, pod := range w.Pods {
-			if pod.HasIstioSidecar() && config.Get().ExternalServices.Istio.IstioAPIEnabled {
+			if pod.HasIstioSidecar() && !w.IsGateway() && config.Get().ExternalServices.Istio.IstioAPIEnabled {
```

Comment:
To be honest, I don't know why `pod.HasIstioSidecar()` is true for a Gateway workload, because a Gateway doesn't really have a sidecar, it is itself a proxy.  If `pod.HasIstioSidecar()` returned false for this case we would not need to remember to add this additional test in various places.

Reply:
`pod.HasIstioSidecar()` is true for the Gateway deployed outside of the 'istio-system' namespace and has 'sidecar.istio.io/inject: "true"'.